themes/custom/az_barrio/az_barrio.theme

```diff
@@ -353,18 +349,18 @@ function az_barrio_az_bootstrap_assets_path($type) {
 /**
  * Helper function for constructing UA Brand Icons asset paths.
  */
-function az_barrio_brand_icons_assets_path($type) {
-  $az_brand_icons_path = '';
+function az_barrio_az_icons_assets_path($type) {
+  $az_icons_path = '';
```

Comment:
@trackleft Do you think we want to consider adding a local copy of the az-icons to Quickstart?  If not, maybe we should just remove this function altogether?

Reply:
I think we should